### DISCUSSION TITLE
Testing Unsupported NIC on single node cluster

### DISCUSSION THREAD

**Discussion by jooshd** [k8snetworkplumbingwg/sriov-network-operator#458] [Q&A]
I'm working with a BCM5709C on a single node cluster, and I recognize that it is unsupported. Since it is unsupported, I enabled DEV_MODE=TRUE in the command line before running make deploy-setup-k8s. The operator is not able to see the NIC based on the logs of the operator pod since it's periodically updating and saying there's nothing to update and there isn't a pod for config-daemon. My question is do I have to also add the NIC to the config map? If so, where could I find information on the vender ID as well as PF and VF device IDs. Thanks!

**Comment by zeeke** [comment]
Hi @jooshd, if you don't see any config-daemon pod, there might be an issue with node labels and daemonset selector. Please, double check if the node matches the `sriov-network-config-daemon` `DaemonSet.Spec.template.spec.nodeSelector`.

**Reply by jooshd** [reply]
Hi @zeeke. Where can I find  sriov-network-config-daemon DaemonSet.Spec.template.spec.nodeSelector? I created a single node cluster with kubespray and assigned it the worker label so the daemonset would see it. 
Tutorial: https://eminaktas.medium.com/single-node-k8s-cluster-with-kubespray-49588410d9d4

**Reply by zeeke** [reply]
I'm talking about this
https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/#running-pods-on-select-nodes

If the daemonset doesn't spawn any pods, there should be a problem with the nodeSelector. 
Please share the output of

```
kubectl get daemonset/sriov-network-config-daemon -o yaml
kubectl get nodes -o yaml
```
